src/controllers/auth.js

```diff
@@ -24,11 +58,15 @@ class AuthController extends BaseController {
    * @param  {string}   token     The jwt token to check
    * @return {Promise|*|PromiseLike<T>|Promise<T>}
    */
-  checkToken (token) {
+  checkToken (token = undefined) {
```

Comment:
The default parameter is useless: it'll set token to undefined only if it is... undefined.